### ISSUE TITLE
A11y_AzureDataStudio_AddNewController_Keyboard : Loss of keyboard focus occur after the "Cancel" button.

### ISSUE THREAD

**Issue by 12-shweta** [12-shweta/Shweta#2]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#MAS2.4.7;DesktopApp;#electron.0,#Keyboard

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tool Name : Keyboard
MAS References : MAS2.4.7

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Connect to the servers with the required credentials.
3. Navigate to the "SQL Server Big Data Cluster" and expand it.
4. Navigate to the "SQL Server Big Data Cluster controller" and click on it.
5. Click on the "Add New Controller" using mouse and navigate till "Cancel" button.
6. Observe that after the "Cancel" button the keyboard focus is not visible .

**Actual:**
After the "Cancel" button , the keyboard focus is not visible on any control after pressing first tab. Pressing tab again the keyboard focus is visible on background.

**Expected:**
After the "Cancel" button , the keyboard focus should be visible on the "URL" edit box.

**User Impact:**
The keyboard only users will get confused while navigating the screen.

**Recommendations:​**
Please refer to below URL which is repository for accessibility bug fixes.​
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](url)

**​MAS Reference:​
MAS2.4.7-**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}](url)

**Attachment for Reference:**
[2_A11y_AzureDataStudio_AddNewController_Keyboard.pptx](https://github.com/12-shweta/Shweta/files/3707159/2_A11y_AzureDataStudio_AddNewController_Keyboard.pptx)

**Comment by 12-shweta** [comment]
By mistake logged  this bug.
